### DISCUSSION TITLE
project admin not active

### DISCUSSION THREAD

**Discussion by vandana79039** [sanjay-kv/Open-source-Practice#833] [Q&A] [answered]
I sent text to assign an issue. but the Project not responded, its been more than 24 hrs. 
Project name: Food_ordering_website

**Comment by PRAYAG0908** [comment]
added that repository url in https://github.com/GSSoC24/Contributor/discussions/94 gssoc will flag that admin

**Comment by aditi75432** [comment, marked as answer]
Wait for some more time .Ensure that your issue is not some some text changes .

**Comment by sanjay-kv** [comment]
this project marked to remove
